test/views/after_auth_screens/events/create_event_page_test.dart

```diff
@@ -354,52 +354,52 @@ void main() {
     expect((tester.widgetList(switches).toList()[1] as Switch).value, false);
   });
 
-  testWidgets("Checking tap Inkwell work for admin list", (tester) async {
```

Comment:
@Kushalrock Can you write similar test case (Inkwell work) for members list (if any) ?

Reply:
It was already written and works fine